README.md

```diff
@@ -199,108 +102,79 @@ $ sudo ./setup_host_rpi.sh
+<details>
+<summary>MacOS (Not Recommended)</summary>
+<br>
 
-# Flash Nocturne image, without resetting the data partition 
-$ sudo python3 ./superbird_tool.py --dont_reset --restore_device /path/to/nocturne-image/image
-```
-After the flashing completes, unplug and replug your Car Thing, and change directories to the scripts folder.
+TBD
 
-```bash
-# Go into the setup-scripts folder
-$ cd /path/to/nocturne-image/setup-scripts
-```
+</details>
 
-There are two different ways to use Nocturne. You can either use it at your desk, or in your car. 
+<br>
 
 <details>
-<summary>Using Nocturne at your desk</summary>
+<summary>Linux (Not Recommended)</summary>
 <br>
 
-At this point, there are two different scripts that you can use. The first one, `setup_host_apt.sh`, is used on Linux distros that utilize apt as it's package manager. The second one, `setup_host_pacman.sh`, is used on Linux distros that utilize Pacman as it's package manager. If you use Pacman, replace `setup_host_apt.sh` in the following commands with `setup_host_pacman.sh` to continue with setup.
-
-It is important to note that `setup_host_pacman.sh` has not been tested very well, so it may not work as intended for setup. 
+TBD
 
-``` bash
-# Make setup_host_apt.sh executable
-$ chmod +x setup_host_apt.sh
-
-# Execute setup_host_apt.sh
-$ sudo ./setup_host_apt.sh
-```
 </details>
 
+<br>
+
 <details>
-<summary>Using Nocturne in your car</summary>
+<summary>Windows (Not Officially Supported)</summary>
 <br>
-To use Nocturne in your car, you will need to have a Raspberry Pi to provide network.  
 
-Connect your Raspberry Pi to your computer, your Car Thing to the Raspberry Pi, and run the following from your command line:
+NOTE: NO SUPPORT IS PROVIDED FOR THIS METHOD
 
-```bash
-# Transfer setup_host_rpi.sh to Raspberry Pi
-$ scp /path/to/nocturne-image/setup-scripts/setup_host_rpi.sh pi@raspberrypi.local:/home/pi/
+NOTE: This setup method is not recommended as there are frequent issues with the AMD USB chipset and recognizing the CarThing. Proceed with caution knowing that it may not work with your PC.
 
-# SSH into Raspberry Pi
-$ ssh pi@raspberrypi.local
+Enter the following commands in powershell as Administrator to allow internet access to your CarThing:
 
-# Make setup_host_rpi.sh executable
-$ chmod +x /home/pi/setup_host_rpi.sh
+```bash
+#Identify correct network adapter is present:
+$ctNic = (Get-NetAdapter -InterfaceDescription "*NDIS*")
 
-# Execute setup_host_rpi.sh
-$ sudo ./setup_host_rpi.sh
+#Set IP address of network interface:
+$ctNic | Set-NetIPAddress -IPAddress 192.168.7.1 -PrefixLength 24
+
+#Allow sharing of network connection to CarThing
+New-NetNat -Name "CarThing" -InternalIPInterfaceAddressPrefix 192.168.7.0/24
 
-# Reboot Raspberry Pi
-$ sudo reboot
 ```
+
+As an FYI, your mileage may vary greately here. You may need to configure the Windows Firewall to allow this traffic depending on your environment. 
+
 </details>
+
 <br>
```

Comment:
Have you tested this windows method? I also would need input from the team on whether we want to bother mentioning this method, as it implies some level of support (even if you say that it's unsupported).

Reply:
Yes I have this working, even on an AMD chip with the bad USB chipset. Occasionally it takes a few powercycles for it to be recognized. But it works a majority of the time. New AMD and Intel platforms have no issues. 

See below:

![20241220_112547](https://github.com/user-attachments/assets/19d7e1e5-1a39-4284-8acc-80cfd19eb9e3)

Reply:
I'll remove this comment from my review for now and I'll ask the team to check on it later on.